opdracht1.js

```diff
@@ -0,0 +1,18 @@
+const tvTypes = inventory.map((tvType) => {
+    return tvType.type;
+})
+console.log(tvTypes);
+
+const tvsSoldOut = inventory.filter((tvSoldOut) => {
+    if (tvSoldOut.sold - tvSoldOut.originalStock === 0)
+        return tvSoldOut;
+})
+console.log(tvsSoldOut);
+
+const tvsWithAmbiLight = inventory.filter((tvWithAmbiLight) => {
+    return tvWithAmbiLight.options.ambiLight === true;
+})
```

Comment:
`boolean_waarde === true` is altijd gelijk aan `boolean_waarde`, dus in dit geval
```suggestion
const tvsWithAmbiLight = inventory.filter((tvWithAmbiLight) => {
    return tvWithAmbiLight.options.ambiLight;
})
```